docker/config/grafana/provisioning/dashboards/lido_basset/lido_basset.json

```diff
@@ -189,15 +433,15 @@
           "exemplar": true,
           "expr": "update_global_index_failed_tx_since_last_check{}",
           "interval": "",
-          "legendFormat": "",
+          "legendFormat": "failed txs since last check",
           "refId": "A"
         },
         {
           "exemplar": true,
           "expr": "update_global_index_successful_tx_since_last_check{}",
           "hide": false,
           "interval": "",
-          "legendFormat": "",
+          "legendFormat": "uusd fee",
```

Comment:
Same as above. Messed up query expr and legend name.

Reply:
done